content/blog/sonata-operator.md

```diff
@@ -1,6 +1,63 @@
 ---
 date: 2024-02-19
 title: What is Sonataflow Operator? 
+# Content is copied from https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/README.md?raw=true
+# and edited to solve broken links
 ---
+# SonataFlow Operator
 
-{{< remoteMD "https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/README.md?raw=true" >}}
```

Comment:
I see the reasons behind this change (from PR description) but copy&paste will make the guide quickly outdated. Can't we just add a link without rendering it? Having a specific page only for the link is too much, maybe we can add the link to another page/section.

Reply:
TBH, there isn't much content in this page that may become outdated.
The only risk I can see there is the api version (`v1alpha08`), but that will become relevant only if someone wishes to develop for the sonataflow in which case they will use the sonataflow-operator repository for that.
I was looking to point to other doc, but other README files are templates which rendered not as good as this one, e.g. https://github.com/apache/incubator-kie-kogito-docs/blob/main/serverlessworkflow/modules/ROOT/pages/cloud/index.adoc

I don't have any strong opinion towards any direction.